### DISCUSSION TITLE
[Bug]: Arknights替身不能启动

### DISCUSSION THREAD

**Discussion by Ricardo-F-X** [MaaAssistantArknights/MaaAssistantArknights#4692] [Q&A] [answered]
Environment
MacOS 13.4 Beta版(22F5059b)

Describe the bug

通过fork版本的playcover安装了Arknights；
但是通过这个方式安装的Arknights不能通过创建替身，拖入应用程序目录的方式 快捷启动。
启动会出现窗体并闪退。
但是同一方式的其他IPA应用均可使用该方式替身启动。

Steps to reproduce

通过fork version的playcover安装IPA游戏；
右键 在finder中打开 创建替身
（可选）将替身移动至 应用程序文件夹
运行替身
出现窗体并闪退
Expected behaviour

通过fork version的playcover安装IPA游戏；
右键 在finder中打开 创建替身
通过替身成功运行IPA游戏。

**Comment by hguandl** [comment, marked as answer]
明日方舟由于使用 OpenGL 库，无法直接启动。需要引入链接库相关的环境变量，例如在终端中执行 `open --env DYLD_LIBRARY_PATH=/usr/lib/system/introspection ~/Library/Containers/io.playcover.PlayCover/明日方舟.app`。如果想从启动台打开的话，可以创建一个自动操作执行上述命令。
